dropwizard-auth/src/main/java/io/dropwizard/auth/CachingAuthorizer.java

```diff
@@ -21,45 +21,6 @@
 
 import static com.codahale.metrics.MetricRegistry.name;
 
-class AuthorizationContext<P extends Principal> {
```

Comment:
This was a top-level class defined in a different java file for some reason. I've extracted it to the top-level package to make it easier to find where this code is found.

It makes sense to me to either package as an inner class or a top-level class in a separate .java file, but not in the same .java file as another class.